### PR TITLE
new script for UnixCluster autogroup fixups (SOFTWARE-5057)

### PR DESCRIPTION
**Pull request by edquist**
See SOFTWARE-5057 for discussion.

### CODE REVIEW

**Comment by edquist** [comment]
(accidentally put blin as assignee earlier instead of reviewer)

**Comment by edquist** [comment]
Added `--fix-all` option for fully-automated use, per SOFTWARE-5057.

**Review by matyasselmeci — DISMISSED** [review]

File: group_fixup.py [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+def get_datalist(listname):
+    def get(data):
+        return data[listname] if data else []
+    return rorable(get)
+
+
+# api call results massagers
+
+
+def get_unixcluser_autogroups():
```
Comment (matyasselmeci) [inline group_fixup.py]:
```suggestion
def get_unixcluster_autogroups():
```

File: group_fixup.py [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+    oldname = group["Name"]
+    newname = get_fixed_unixcluster_group_name(oldname)
+    if oldname != newname:
+        print('  ** Rename group to: "%s"' % newname)
+    show_group_identifiers(group["Id"])
+    print("")
+
+
+def show_all_unixcluster_groups():
+    groups = get_unixcluser_autogroups()
```
Comment (matyasselmeci) [inline group_fixup.py]:
```suggestion
    groups = get_unixcluster_autogroups()
```

**Review by matyasselmeci — APPROVED** [review]
Two fixups, otherwise pre-approving.

**Pull request merged**